src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -315,6 +319,35 @@ void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        void BeginDuplicateConnection(Guid nodeId, int portIndex, PortType portType)
```

Comment:
please mark this with an access modifier.

Reply:
interestingly almost none of the methods `DynamoModelCommands.cs` have access modifiers, I guess because they are all expected to be `internal`?

Reply:
`internal` seems reasonable

Reply:
I think if not specific the default is actually private.

Reply:
I differs if it is interface or class or methods, see https://stackoverflow.com/questions/2521459/what-are-the-default-access-modifiers-in-c